pycytominer/cyto_utils/util.py

```diff
@@ -124,18 +168,83 @@ def check_consensus_operation(operation):
     return operation
 
 
-def get_pairwise_correlation(population_df, method="pearson"):
+def check_fields_of_view_format(fields_of_view):
+    """Confirm that the input fields of view is valid.
+
+    Parameters
+    ----------
+    fields_of_view : list of int, default "all"
```

Comment:
This does not currently default to "all" - either delete this comment, or fix function arg

Reply:
https://github.com/cytomining/pycytominer/pull/129/commits/6bfee56de80393a31ca93fd139bae6265fb7733f